nex/register_common_secure_server_protocols.go

```diff
@@ -30,15 +30,18 @@ func registerCommonSecureServerProtocols() {
 
 	matchMakingProtocol := match_making.NewProtocol()
 	globals.SecureEndpoint.RegisterServiceProtocol(matchMakingProtocol)
-	common_match_making.NewCommonProtocol(matchMakingProtocol)
+	commonMatchMakingProtocol := common_match_making.NewCommonProtocol(matchMakingProtocol)
+	commonMatchMakingProtocol.SetManager(globals.MatchmakingManager)
 
 	matchMakingExtProtocol := match_making_ext.NewProtocol()
 	globals.SecureEndpoint.RegisterServiceProtocol(matchMakingExtProtocol)
-	common_match_making_ext.NewCommonProtocol(matchMakingExtProtocol)
+	commonMatchMakingExtProtocol := common_match_making_ext.NewCommonProtocol(matchMakingExtProtocol)
+	commonMatchMakingExtProtocol.SetManager(globals.MatchmakingManager)
 
 	matchmakeExtensionProtocol := matchmake_extension.NewProtocol()
 	globals.SecureEndpoint.RegisterServiceProtocol(matchmakeExtensionProtocol)
 	commonMatchmakeExtensionProtocol := common_matchmake_extension.NewCommonProtocol(matchmakeExtensionProtocol)
+	commonMatchmakeExtensionProtocol.SetManager(globals.MatchmakingManager)
 
 	commonMatchmakeExtensionProtocol.CleanupSearchMatchmakeSession = nex_matchmake_extension_common.CleanupSearchMatchmakeSession
```

Comment:
We also need a `CleanupMatchmakeSessionSearchCriterias` for Matchmaking to work in this game.

Reply:
Are there any specifics on what this needs to do? Or would stubbing it work

Reply:
Stubbing works, the method just throws an error if it isn't handled at all

Reply:
If a stub works then we should update common at some point to either provide a default or check if this function is set